pages/k8s/install-aws.md

```diff
@@ -0,0 +1,282 @@
+---
+wrapper_template: "base_docs.html"
+markdown_includes:
+  nav: "shared/_side-navigation.md"
+context:
+  title: "CDK on AWS"
+  description: Running CDK on AWS using the aws-integrator.
+keywords: aws, integrator, ebs, elb
+tags: [install]
+sidebar: k8smain-sidebar
+permalink: using-aws.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+The **Charmed Distribution of Kubernetes<sup>;&reg</sup>** will run seamlessly on
```

Comment:
`&reg` is missing a `;`

Reply:
Actually, I guess the `;` is just in the wrong place.

Reply:
yeah. there were a few other tweaks as well

Reply:
thanks for looking!